yarn.lock

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
```

Comment:
bajillion changes due to https://github.com/jest-community/jest-junit/pull/80

Reply:
Can you do this in a different PR please?

Reply:
this is just running `yarn` - you'll get the same result running `yarn` on master

Reply:
"bajillion changes due to" was meant to refer to the fact `jest-junit` uses Jest 23 so now that we are on Jest 24, yarn needs to install Jest 23 `config` and `validate`